### PR TITLE
chore(wg-ecosystem): update list of associated repositories

### PR DESCRIPTION
**Pull request by HashimotoYT**
Ref https://electronhq.slack.com/archives/CNTJBD467/p1596051064254800

Updates the list of associated repositories with that what's we have inside `.permissions`. This be done automatically so need some data validation checking

### CODE REVIEW

File: wg-ecosystem/README.md [hunk 1]
```diff
@@ -63,22 +63,38 @@ These repos are sorted alphabetically, their order does not reflect that any of
...
 * [`electron/get`](https://github.com/electron/get)
+* [`electron/hubdown`](https://github.com/electron/hubdown)
 * [`electron/i18n`](https://github.com/electron/i18n)
+* [`electron/node-minidump`](https://github.com/electron/node-minidump)
 * [`electron/node-rcedit`](https://github.com/electron/node-rcedit)
 * [`electron/rcedit`](https://github.com/electron/rcedit)
 * [`electron/releases`](https://github.com/electron/releases)
+* [`electron/remote`](https://github.com/electron/remote)
+* [`electron/semantic-release-npm-cfa`](https://github.com/electron/semantic-release-npm-cfa)
+* [`electron/simple-samples`](https://github.com/electron/simple-samples)
```
Comment (erickzhao) [inline wg-ecosystem/README.md:93]: Should we hide this because the repo has been archived?

File: wg-ecosystem/README.md [hunk 2]
```diff
@@ -63,22 +63,38 @@ These repos are sorted alphabetically, their order does not reflect that any of
...
 
+* [`electron/algolia-indices`](https://github.com/electron/algolia-indices)
 * [`electron/apps`](https://github.com/electron/apps)
+* [`electron/asar`](https://github.com/electron/asar)
+* [`electron/docs-parser`](https://github.com/electron/docs-parser)
+* [`electron/download-stats`](https://github.com/electron/download-stats)
 * [`electron/electron-api-demos`](https://github.com/electron/electron-api-demos)
+* [`electron/electron-api-historian`](https://github.com/electron/electron-api-historian)
 * [`electron/electron-docs-linter`](https://github.com/electron/electron-docs-linter)
+* [`electron/electron-docs`](https://github.com/electron/electron-docs)
```
Comment (erickzhao) [inline wg-ecosystem/README.md:74]: nit: electron-docs should go in front of electron-docs-linter for alphabetical order

File: wg-ecosystem/README.md [hunk 3]
```diff
@@ -63,22 +63,38 @@ These repos are sorted alphabetically, their order does not reflect that any of
...
+* [`electron/electron-issues`](https://github.com/electron/electron-issues)
 * [`electron/electron-notarize`](https://github.com/electron/electron-notarize)
 * [`electron/electron-osx-sign`](https://github.com/electron/electron-osx-sign)
 * [`electron/electron-packager`](https://github.com/electron/electron-packager)
+* [`electron/electron-quick-start-typescript`](https://github.com/electron/electron-quick-start-typescript)
```
Comment (erickzhao) [inline wg-ecosystem/README.md:79]: nit: electron/electron-quick-start should go above electron/electron-quick-start-typescript in alphabetical order

File: wg-ecosystem/README.md (hunk 1 above)
Comment (HashimotoYT) [reply wg-ecosystem/README.md:93]: That repo still have an entry in `.permission` about that `wg-ecosystem` have to write access, I can add logic for skip archive repos if we prefer to not show them, or just remove `ecosystem-wg` access from `.permissions`  what's seems to prefer for me

Comment (erickzhao) [reply wg-ecosystem/README.md:93]: Sounds good to me! In any case, write access to an archived repo means nothing without admin permissions. 🤷

**Comment by HashimotoYT** [comment]
List updated with the new `search-with-your-keyboard` repo and removing the old archived repos

**Review by erickzhao — APPROVED** [review]

**Pull request merged**